css/types/image.json

```
@@ -1639,24 +1639,46 @@
                 "version_added": false
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": false
               },
               "edge": {
-                "version_added": null
+                "version_added": false
               },
               "edge_mobile": {
-                "version_added": null
-              },
-              "firefox": {
-                "prefix": "-moz-",
-                "version_added": "4",
-                "notes": "<code>element()</code> is limited to <a href='https://developer.mozilla.org/docs/Web/CSS/background-image'><code>background-image</code></a> and <a href='https://developer.mozilla.org/docs/Web/CSS/background'><code>background</code></a>."
-              },
-              "firefox_android": {
-                "prefix": "-moz-",
-                "version_added": "4",
-                "notes": "<code>element()</code> is limited to <a href='https://developer.mozilla.org/docs/Web/CSS/background-image'><code>background-image</code></a> and <a href='https://developer.mozilla.org/docs/Web/CSS/background'><code>background</code></a>."
+                "version_added": false
               },
+              "firefox": [
+                {
+                  "prefix": "-moz-",
+                  "version_added": "57"
+                },
+                {
+                  "prefix": "-moz-",
```

This is a great update. Seeing this spelled out now, I think we probably need to add this:

```suggestion
                  "prefix": "-moz-",
                  "partial_implementation": true,
```

Sorry for not realizing this sooner!